src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/preferences/EditPreferencesPage.java

```diff
@@ -30,7 +25,7 @@ public EditPreferencesPage openEditingSection() {
     return openEditingSection(urlBuilder.getUrlForWiki(Configuration.getWikiName()));
   }
 
-  public EditPreferencesPage openEmailSection(String wikiURL) {
+  private EditPreferencesPage openEmailSection(String wikiURL) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "openEmailSection" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

Reply:
it is used